flair/datasets/sequence_labeling.py

```diff
@@ -1508,6 +1508,308 @@ def __init__(
+
+                        if words > 1: # gather single words of chunk
+
+                            is_chunk = True
+
+                            # save single words of chunk
+                            chunk_parts = []
+
+                            # handle first word of chunk
+                            word_start = 0
+                            word_end = chunk.find('_', word_start)
+                            f.write(chunk[word_start:word_end])
+                            word_start = word_end+1
+
+                            for _ in range(words-1):
+
+                                word_end = chunk.find('_', word_start)
+
+                                if word_end == -1:
+
+                                    chunk_parts.append(chunk[word_start:])
+
+                                else:
+
+                                    chunk_parts.append(chunk[word_start:word_end])
+
+                                word_start = word_end+1
+
+                        else:
+
+                            f.write(chunk)
+
+                        ctr += 1
+                        continue
+
+                    elif 'pos=' in token:
+
+                        if ctr != 2:
+
+                            temp.append(' O')
+                            f.write(' O')
+
+                        add_tag(token)
+                        ctr = 3
+                        continue
+
+                    elif '"' in token:
+
+                        add_tag(token)
+                        ctr += 1
+                        continue
+
+                    else:
+
+                        # edit here for variation
+                        for _ in range(4-ctr):
+
+                            temp.append(' O')
+                            f.write(' O')
+
+                        f.write('\n')
+
+                if is_chunk: # handle chunks
+
+                    for word in chunk_parts:
+
+                        f.write(word)
+
+                        for elem in temp:
+
+                            if ' O' in elem:
+
+                                f.write(elem)
+
+                            else:
+
+                                f.write(' I-' + elem)
+
+                        f.write('\n')
+
+            elif line_list[0] == '</sentence>':  # handle end of sentence
+
+                f.write('\n')
+
+
+class WSD_UFSAC(ColumnCorpus):
+    def __init__(
+            self,
+            base_path: Union[str, Path] = None,
+            in_memory: bool = True,
+            document_as_sequence: bool = False,
+    ):
+        """
+        Initialize a custom corpus with any two WSD datasets in the UFSAC format. This is only possible if you've
+        manually downloaded the WSD datasets in UFSAC format to your machine.
+        Obtain the most recent datasets from https://drive.google.com/file/d/1Oigo3kzRosz2VjyA44vpJZ58tDFyLRMO and copy
+        any two of the datasets in a folder called 'wsd_ufsac'. Then rename the files in the folder to train.xml and
```

Comment:
This is inconvenient since this requires me to always rename different files to `train.xml` if I want to train on a different dataset. It would be easier if train and test file names are passed as arguments in the contrunctor.